test/python/primitives/test_backend_sampler.py

```diff
@@ -22,23 +22,24 @@
 from qiskit.circuit.library import RealAmplitudes
 from qiskit.primitives import BackendSampler, SamplerResult
 from qiskit.providers import JobStatus
-from qiskit.providers.backend_compat import BackendV2Converter
 from qiskit.providers.basic_provider import BasicSimulator
-from qiskit.providers.fake_provider import Fake7QPulseV1, GenericBackendV2
+from qiskit.providers.fake_provider import GenericBackendV2
 from qiskit.transpiler import PassManager
 from qiskit.utils import optionals
 from test import QiskitTestCase  # pylint: disable=wrong-import-order
 from test import combine  # pylint: disable=wrong-import-order
 from test.python.transpiler._dummy_passes import DummyAP  # pylint: disable=wrong-import-order
+from ..legacy_cmaps import LAGOS_CMAP
 
 
-BACKENDS = [Fake7QPulseV1(), BackendV2Converter(Fake7QPulseV1())]
-
-BACKENDS_V1 = [Fake7QPulseV1()]
-BACKENDS_V2 = [
-    BackendV2Converter(Fake7QPulseV1()),
+BACKENDS = [
```

Comment:
Also here: this is just a singleton now so would be nice to clean up the `@combine` usage below